### ISSUE TITLE
No interwiki links with Wikipedia

### ISSUE THREAD

**Issue by Anton-Latukha** [haskell/haskell-wiki-configuration#21]
There are many interwiki links configured:
https://wiki.haskell.org/Special:Interwiki

But there is no major one - Wikipedia.

I wonder, is there is a particular technical case associated with it.

It would be proper to use interwiki links instead of external links to major MediaWiki resource.

Related information:
  * https://www.mediawiki.org/wiki/Extension:Interwiki
  * https://www.mediawiki.org/wiki/Manual:Interwiki
  * https://wiki.archlinux.org/index.php/Special:Interwiki

**Comment by hgolden** [comment]
I agree with adding an interwiki link to Wikipedia. When I recently upgraded the MediaWiki version I added the Interwiki extension, but I haven't done anything with it. In order to add another interwiki link, a Sysop has to make a change to the table. That's all that is required. Note: I am not a Sysop, though I probably should become one.

There is a default Interwiki table (see https://phabricator.wikimedia.org/source/mediawiki/browse/master/maintenance/interwiki.sql), which includes "wikipedia" as the interwiki link to Wikipedia. I suggest following this convention.

Other interwiki links are welcome, too.

**Comment by Anton-Latukha** [comment]
I can not follow this convention at the moment.
Until particular interwiki links are added to the extension - they do not work (I even just tested).

So that is why I would skip linking to Wikipedia for now.

**Comment by hgolden** [comment]
I understand your point. I would be happy to add the link to Wikipedia, but it needs to be done by a Sysop. I’m not a Sysop at present. I would like the haskell.org committee to approve me as one. (Note: I could make that change myself, but I’d rather get the committee’s approval first. I’m not Grace Murray Hopper!)

Howard

**Comment by Anton-Latukha** [comment]
It would be great if a mass conversion of currently present Wikipedia links to interwiki links would happen.

But for that, the relevant server-side scripting must be configured.

**Comment by Anton-Latukha** [comment]
Oops, sorry, I rerolled the message, but you use email.
I believe then it is OK if I as a newcomer to HaskellWiki post a couple observed things to ask about.

**Comment by Anton-Latukha** [comment]
[Just found](https://github.com/haskell/haskell-wiki-configuration/issues/23#issuecomment-481203598) that that current `wiki-monkey` can convert external links to proper interwiki links.
And it can and does much more.

And it only the beginning of possibilities.

**Comment by Anton-Latukha** [comment]
List of interlink seems really old.

It can have a way of partial fix. 
Or:
1. Get the modern default list.
2. Match name&adress in HaskellWiki and modern default list.
3. Unique records to HaskellWiki list match to provided interwiki link counts stats in the engine.
4. Run shell loop on unique records to HaskellWiki, pinging addresses, getting default webpage with curl, to find what of those are still online. From the knee:
  ```
  # live impromptu on https://unix.stackexchange.com/a/84820
  for line in $(cat ./file.list)
  do
    responce="$(echo $line | awk '{print $2} | xargs -L1 curl -Is)"
    echo $responce | head -n 1
  done
  ```
5. Delete unused/used one-two times links, delete unresponded links - they do not work anyway. Merge new list with unique, used and responsive urls from the old HaskellWiki list.
